src/lwc/geTemplateBuilderWidget/geTemplateBuilderWidget.html

```diff
@@ -10,6 +10,13 @@
                         <p class="slds-text-body_regular widget-text">{CUSTOM_LABELS.commonPercent}</p>
                     </lightning-layout-item>
                 </template>
+                <template if:true={isSoftCredit}>
+                    <lightning-layout-item size="12">
+                        <label class="slds-text-body_regular widget-label slds-p-bottom_xx-small">{uiTitle}</label>
+                        <p class="slds-text-body_regular widget-text">{ocrContactFieldApiName}</p>
+                        <p class="slds-text-body_regular widget-text">{ocrRoleFieldApiName}</p>
```

Comment:
Looks like we may actually want the field labels here vs the api name. Just a note, I don't think this critical for this PR.